tests/pipeline/test_downsampling.py

```diff
@@ -684,3 +689,80 @@ def test_errors_on_bad_downsample_frequency(self):
             "for argument 'frequency', but got 'bad' instead."
         ).format(_qualified_name(f.downsample))
         self.assertEqual(str(e.exception), expected)
+
+
+class TestDownsampledRowwiseOperation(WithAssetFinder, ZiplineTestCase):
+    T = partial(pd.Timestamp, tz='utc')
+    START_DATE = T('2014-01-01')
+    END_DATE = T('2014-02-01')
+    HALF_WAY_POINT = T('2014-01-15')
+
+    dates = pd.date_range(START_DATE, END_DATE)
+    factor = TestingDataSet.float_col.latest
+
+    @classmethod
```

Comment:
yeah, I had to manually change it to get the test to fail correctly on master. I made some comments on your pr.